### PR TITLE
Some tweaks to the contributing doc

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Things you will need
 
  * Linux or Mac OS X. (Windows is not yet supported.)
  * git (used for source version control).
- * An IDE. We recommend [IntelliJ with the Flutter plugin](https://flutter.io/intellij-ide/).
+ * An IDE. We recommend [IntelliJ with the Flutter plugin](https://flutter.io/intellij-ide/) or Xcode.
```
Comment (abarth) [inline CONTRIBUTING.md]: IntelliJ and Xcode are in different categories.  IntelliJ is what we recommend for editing the Dart code in the flutter.git repository.  Xcode is a tool that's required in order to build apps for iOS.

Comment (xster) [inline CONTRIBUTING.md]: This is engine.git :)
I meant using Xcode to edit objective-c/c++ code.

Comment (abarth) [inline CONTRIBUTING.md]: Ah, i was confused.

**Comment by abarth** [comment]
LGTM

**Pull request merged**